doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -9,22 +9,93 @@
+
+
 
 <a name="3.2"></a>
 ## 3.2 Access and Connectivity
 
-> Describe the access and connectivity requirements for any lab used to deploy an RI.
+This section describes the access and connectivity requirements for labs used to deploy an RI.
+
+
+<a name="3.2.1"></a>
+### 3.2.1 Lab Access
+
+The requirements on and means of accessing a given lab differ per hosting organization and use case:
+
+* **Community labs**, such as those hosted by OPNFV, must provide remote access from the Internet to allow usage of the lab resources by community members. Remote access shall be managed and secured by means of VPN or SSH gateways. Both the public and out-of-band management networks of the lab shall be accessible from the VPN / SSH gateway host.
+
+* Access to lab resources hosted by **public cloud providers** is managed by the cloud provider. If a public cloud provider is chosen to host community-based development resources, access to those resources shall be available to all community members by means of a application process. 
+
+* Access to **internal development and test labs** of software vendors, operators and 3rd party test labs is managed by the corresponding entity and not expected to be available to community members outside of the respective organization.
+
+
+<a name="3.2.2"></a>
+### 3.2.2 Connectivity
+
+The network fabric in the lab must support the following networks:
+
+* Out-of-Band hardware management
+* Admin (host OS access)
+* Public (external / Internet facing network)
+* Private (cluster internal network)
+* Storage
+
+The network topology should provide at least 2 networks with preallocated IP addressing schemes for the out-of-band management network and the Public network. The Public network must be able to reach / access the public Internet, preferably directly or via an enterprise proxy. At least IPv4 addresses must be available, ideally IPv6 address space should also be supported. At least 1 network interface must be capable of performing PXE boot and that network must be available to both the jump / test host and each physical server.
+
+The network fabric shall comprise redundant layer 2 connectivity between hosts. The switching hardware shall support VLANs for separating networks as well as supporting line speeds corresponding to the speeds of the NICs listed below. 
+
 
 <a name="3.3"></a>
 ## 3.3 Infrastructure Requirements
 
-> Describe the requirements on any infrastructure used to deploy the RI onto (i.e. from the HW profile and SW profile specifications in RA2).
+The requirements on infrastructure hardware used to deploy and run the RIi2 are as follows:
+
+1. One (1) physical server dedicated as a jump / test host
+1. Six (6) physical servers, serving as either controller or compute/worker nodes
```

Comment:
Do they need to be physical? i.e. would VMs be sufficient? I think we could allow either physical or virtual servers?

Reply:
yes, you are right.